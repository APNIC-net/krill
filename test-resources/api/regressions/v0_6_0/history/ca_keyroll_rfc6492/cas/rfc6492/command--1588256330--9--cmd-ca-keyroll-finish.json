{
  "actor": "krill",
  "time": "2020-04-30T14:18:50.538608Z",
  "handle": "rfc6492",
  "version": 12,
  "sequence": 9,
  "details": {
    "key_roll_finish": "0"
  },
  "effect": {
    "events": [
      12
    ]
  }
}