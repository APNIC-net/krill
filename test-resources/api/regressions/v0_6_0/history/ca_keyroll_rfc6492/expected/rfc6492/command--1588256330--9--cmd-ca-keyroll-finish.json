{
  "command": {
    "actor": "krill",
    "time": "2020-04-30T14:18:50.538608Z",
    "handle": "rfc6492",
    "version": 12,
    "sequence": 9,
    "details": {
      "key_roll_finish": "0"
    },
    "effect": {
      "events": [
        12
      ]
    }
  },
  "result": {
    "Events": [
      {
        "id": "rfc6492",
        "version": 12,
        "details": {
          "key_roll_finished": [
            "0",
            {
              "ca_repo": "rsync://localhost/repo/rfc6492/0",
              "added": [],
              "updated": [],
              "withdrawn": [
                {
                  "name": "5D3CDDC79F0F00FD59E1312472F996057878CF76.crl",
                  "hash": "417fce400ef03e410f268f96e11d71fea513a489f5f282abfc67269515c92dc7"
                },
                {
                  "name": "5D3CDDC79F0F00FD59E1312472F996057878CF76.mft",
                  "hash": "6c2306367efa401bf2de956effec0a49f2928cb5d28a8a95d957b624684f01a0"
                }
              ]
            }
          ]
        }
      }
    ]
  }
}